lms/djangoapps/bulk_email/tasks.py

```diff
@@ -408,6 +408,11 @@ def _get_source_address(course_id, course_title):
         re.sub(r"[^\w.-]", '_', course_id.course),
         settings.BULK_EMAIL_DEFAULT_FROM_EMAIL
     )
+
+    # make sure it's not longer than 320 chars
+    if len(from_addr) >= 320:
```

Comment:
I'm not sure how truncation works in other languages. This from_addr should really be translated (see #11055 which got stuck because I wasn't sure how to verify the change).

@adampalay if you know how to verify this change and want to fold in #11055, that would be awesome!